zipkin-ui/test/component_data/trace.test.js

```diff
@@ -453,8 +453,8 @@ describe('convertSuccessResponse', () => {
       traceId: '1e223ff1f80f1c69',
       duration: '99.411ms',
       services: 2,
-      depth: 3,
-      spanCount: 3,
+      depth: 4,
```

Comment:
look below! see how we used to have serviceNameAndSpanCounts summing to a different count than spanCount!